metadata/metadata.ts

```diff
@@ -2648,12 +2658,56 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
   },
   Pancakes: {
     description:
-      "A great start to a Bumpkins day. You can can cook these at the Kithcen.",
+      "A great start to a Bumpkins day. You can can cook these at the Kitchen.",
     decimals: 0,
     external_url: "https://docs.sunflower-land.com/getting-started/about",
     image_url: "../public/erc1155/images/523.png",
     attributes: [{ trait_type: "Purpose", value: "Food" }],
   },
+  "Apple Juice": {
+    description:
+      "A crisp refreshing beverage. You can can prepare these at the Smoothie Shack.",
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/535.png",
```

Comment:
These images appear to be missing at the moment.

I would suggest using the `brown_background.png` in the repo to generate these images for Open Sea 👍